### DISCUSSION TITLE
Getting Output values for resources deployed with CAF

### DISCUSSION THREAD

**Discussion by Bhagyashree1094** [aztfmod/terraform-azurerm-caf#1495] [General]
Hi Team,

I am deploying CAF terraform resources with AzureDevOps yaml pipeline .

Can someone guide me on how can I get output values for resources deployed .

Thanks,
Bhagyashree

**Comment by TjommeVergauwen** [comment]
Using
`locals {
   CAFOutput = tomap(
    { "CAFOutput" = {
      #      for key, value in module.caf.storage_accounts[*] : key => value
      for key, value in module.caf : key => value
      if try(value, {}) != {}
      }
    }
  )
}`
you can create a map of the output generated by the CAF mudule.
This cdata can be written to a file for example to get an idea of what is generated.
However, the data itself is not usable in a for_each loop as the data is marked as sensitive. No idea how you can fix this. I'm currently facing that issue as I'm generating databricks workspaces with CAF and I need to work with them further outside of CAF due to missing functionality in CAF...
